docs/Using-Fleet/configuration-files/README.md

```diff
@@ -501,9 +501,21 @@ spec:
 > **Warning:** this will potentially log a lot of data. Some of that data might be private, please verify it before posting it
 in a public channel or a GitHub issue.
 
+## Host Expiry Settings
+
+The `host_expiry` section allows you to define if and when hosts should be removed from Fleet if they have not checked in. Once a host has been removed from Fleet, it will need to re-enroll with a valid `enroll_secret` to connect to your Fleet instance. 
```

Comment:
```suggestion
The `host_expiry` section lets you define if and when hosts should be removed from Fleet if they have not checked in. Once a host has been removed from Fleet, it will need to re-enroll with a valid `enroll_secret` to connect to your Fleet instance. 
```